jsa/features/people.feature

```diff
@@ -64,13 +64,13 @@ Feature: People
     And I should see "People" in the "First sidebar" region 
     And I should see "Contact and Location" in the "First sidebar" region
 
-  @content @live
+  @content @deploy
   Scenario: Staff layout with faculty
     Given I am on "people/staff/faculty"
     Then I should see "Faculty" in the "Content Head" region 
     And I should see "Jacob Smith" in the "Content Body" region
 
-  @content @live
+  @content @deploy
   Scenario: Faculty layout with staff
     Given I am on "people/faculty/staff"
```

Comment:
That's a good question, John. I've been thinking of these as simply additional views for someone like Marion to use when she wants a different layout.